recipes/qwen3.8-flash-next/README.md

```
@@ -0,0 +1,352 @@
+kubectl apply -f model-cache/model-download.yaml -n ${NAMESPACE}
+kubectl wait --for=condition=Complete job/model-download -n ${NAMESPACE} --timeout=7200s
+```
+
+The Job sets `HF_HOME=/model-cache`, so the checkpoint lands in the PVC's Hugging Face cache; each
+worker passes the repo id (`Inferact/Qwen3.8-Flash-Next-NVFP4`) to `--model` and resolves the weights
+from there.
+
+> [!NOTE]
+> The containers run as `runAsUser: 0` because the cached weight files are root-owned while the image
+> defaults to a non-root uid.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)